compiler/optimizations/remoteValueForwarding.cpp

```diff
@@ -70,6 +72,8 @@ void remoteValueForwarding() {
     Map<Symbol*, Vec<SymExpr*>*> defMap;
     Map<Symbol*, Vec<SymExpr*>*> useMap;
 
+    replaceRecordWrappedRefs();
```

Comment:
I made the changes in remoteValueForwarding.cpp because I think we wouldn't perform this transformation unless we were trying to reduce comm-counts by transforming refs into values. I also don't think it's useful to other passes at the moment, so placing it in its own file seems unnecessary.